### ISSUE TITLE
Val and Test loss not matching the training loss

### ISSUE THREAD

**Issue by sachaguer** [sachaguer/jaxili#14]
Validation and Test loss output by the trainer don't match the training loss. The sampling is also incorrect. A normalization step is likely missing in the trainer for validation and test sets.

**Comment by sachaguer** [comment]
Solved. Default value of the seed of ConditionalMAF was `None`. This can create issue. The seed shall never be None.
